src/jobs/jobs.service.ts

```
@@ -1,30 +1,132 @@
+  }
+
+  async removeApplication(id: number): Promise<{ message: string }> {
+    const result = await this.applicationRepository.delete(id);
+    if (result.affected === 0) {
+      throw new NotFoundException(`Application with ID ${id} not found`);
+    }
+    return { message: 'Application removed successfully' };
+  }
+
+  // Weekly analytics for jobs
+  async getWeeklyNewJobsCount(): Promise<Array<{ week: string; count: number }>> {
```

ESLint

[error] Replace `Array<{·week:·string;·count:·number·}>` with `⏎····Array<{·week:·string;·count:·number·}>⏎··`

(prettier/prettier)